frostsnap_core/src/coordinator.rs

```diff
@@ -15,21 +18,69 @@ use schnorr_fun::{
     fun::prelude::*,
     Schnorr, Signature,
 };
-use sha2::{digest::FixedOutput, Digest, Sha256};
+use sha2::Sha256;
 
 use crate::DeviceId;
 
 pub const MIN_NONCES_BEFORE_REQUEST: usize = 5;
 
 #[derive(Debug, Clone, Default)]
 pub struct FrostCoordinator {
-    keys: BTreeMap<KeyId, CoordinatorFrostKey>,
-    key_order: Vec<KeyId>,
+    keys: BTreeMap<Appkey, CoordFrostKey>,
```

Comment:
it *could* be but given that `Appkey` is a unique identifier that callers will refer to the key by it makes sense to represent that in the data structure. Why do we want to avoid `coord_frostkey.appkey`?